src/components/CurrentScenarioSupersetReport/CurrentScenarioSupersetReport.js

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+import React from 'react';
+import { useTranslation } from 'react-i18next';
+import PropTypes from 'prop-types';
+import { Backdrop, Box, CircularProgress, Stack, Typography } from '@mui/material';
+import { ErrorBoundary, SupersetEmbed } from '@cosmotech/ui';
+import { RUNNER_RUN_STATE } from '../../services/config/ApiConstants';
+import { useCurrentScenarioSupersetReport } from './CurrentScenarioSupersetReportHook';
+
+const CurrentScenarioSupersetReport = ({
+  alwaysShowReports,
+  isParentLoading = false,
+  reportConfiguration,
+  iframeRatio,
+  index,
+  labels,
+  ...other
+}) => {
+  const { t } = useTranslation();
+
+  const { currentScenarioData, isSupersetReducerLoading, report, guestToken, options } =
+    useCurrentScenarioSupersetReport();
+
+  const defaultErrorDescription =
+    'Something went wrong when trying to display Superset dashboards. If the problem ' +
+    'persists, please contact an administrator.';
+
+  const showLoadingBackdrop =
+    (currentScenarioData?.state === RUNNER_RUN_STATE.SUCCESSFUL || alwaysShowReports === true) &&
+    isSupersetReducerLoading &&
+    !isParentLoading;
+
+  return (
+    <Box sx={{ height: '100vh', position: 'relative' }}>
+      <Backdrop
+        data-cy="superset-charts-backdrop"
+        open={showLoadingBackdrop}
+        sx={{
+          position: 'absolute',
+          color: '#fff',
+          zIndex: (theme) => theme.zIndex.drawer + 1,
+        }}
+      >
+        <Stack spacing={2} alignItems="center">
+          <CircularProgress data-cy="superset-charts-loading-spinner" size={24} color="inherit" />
+          <Typography data-cy="superset-charts-loading-text" variant="h6">
+            {t(
+              'commoncomponents.iframe.scenario.results.label.chartsLoading',
+              'Requesting access to Superset dashboards...'
```

Comment:
In translation dictionaries, this label mentions PowerBI instead of Superset. The 2nd argument of `t` is just a fallback value to display when the key is not found in the dictionaries.
I suggest we modify the label in the dictionaries and the fallback labels in both components (superset and powerBi) to have: `Requesting access to dashboards...`

Same thing for `defaultErrorDescription`, we could change it to a more generic message in components and in the translation dicts:
```
const defaultErrorDescription =
  'Something went wrong when trying to display dashboards. If the problem ' +
  'persists, please contact an administrator.';
```